superset-frontend/packages/superset-ui-core/test/query/buildQueryObject.test.ts

```diff
@@ -285,7 +285,8 @@ describe('buildQueryObject', () => {
         datasource: '5__table',
         granularity_sqla: 'ds',
         viz_type: 'table',
-        url_params: null as unknown as undefined,
+        // @ts-ignore
```

Comment:
ignore is used for the test case.

Reply:
I believe `// @ts-expect-error` is the correct syntax for testing with incorrect types